tests/wpt/metadata-css/css-animations-1_dev/html/css-filters-animation-brightness.htm.ini

```diff
@@ -1,3 +1,3 @@
 [css-filters-animation-brightness.htm]
   type: reftest
-  expected: FAIL
+  disabled: The reftest of servo with non-expired animations failed(issue 17335).
```

Comment:
There is no need to disable these tests. We should use `expected: TIMEOUT` instead, and add a separate `bug: https://github.com/servo/servo/issues/17335` as well.